src/domain/community/CommunityContributors/CommunityContributorsAccordion.tsx

```diff
@@ -1,30 +1,13 @@
 import { Accordion, AccordionProps } from '../../../components/composite/common/Accordion/Accordion';
-import Typography from '@mui/material/Typography';
 import React from 'react';
-import { useTranslation } from 'react-i18next';
-import { SectionSpacer } from '../../shared/components/Section/Section';
-import { styled } from '@mui/styles';
-import withOptionalCount from '../../shared/utils/withOptionalCount';
-import ContributingOrganizations, { ContributingOrganizationsProps } from './ContributingOrganizations';
-import ContributingUsers, { ContributingUsersProps } from './ContributingUsers';
+import CommunityContributorsView, { CommunityContributorsViewProps } from './CommunityContributorsView';
```

Comment:
I have moved a lot of the contents of this file into the new CommunityContributorsView to reuse the content for the dialog without the Accordion component